mdbenchmark/analyze.py

```diff
@@ -94,12 +93,9 @@ def analyze(directory, plot, ncores, output_name):
     # move this to the bundle function!
     PrintDataFrame(df)
 
-    # here we determine which output name to use.
-    if output_name is None:
-        output_name = generate_output_name("csv")
-    if ".csv" not in output_name:
-        output_name = "{}.csv".format(output_name)
-    df.to_csv(output_name)
+    if save_csv is not None and ".csv" not in save_csv:
```

Comment:
Just realized this will actually allow files with the name `my_output.csv.something`. Instead we should do:

```suggestion
    if save_csv is not None and not save_csv.endswith(".csv"):
```